blockchain/validate.go

```diff
@@ -684,6 +689,12 @@ func compareScript(height int32, script []byte) error {
 func CheckBlockHeaderContext(header *wire.BlockHeader, prevNode HeaderCtx,
 	flags BehaviorFlags, c ChainCtx, skipCheckpoint bool) error {
 
+	// The height of this block is one more than the referenced previous
+	// block.
+	blockHeight := prevNode.Height() + 1
```

Comment:
nit: kinda weird this code block is moved here since `blockHeight` isn't used until line 729